HPS/Heartland/view/frontend/layout/vault_cards_listaction.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~  Heartland payment method model
+~
+~  @category    HPS
+~  @package     HPS_Heartland
+~  @author      Heartland Developer Portal <EntApp_DevPortal@e-hps.com>
+~  @copyright   Heartland (http://heartland.us)
+~  @license     https://github.com/hps/heartland-magento2-extension/blob/master/LICENSE.md
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <update handle="customer_account"/>
+    <head>
+        <script src="https://hps.github.io/token/2.1/securesubmit.js" src_type="url" />
+
+        <!-- Cert
+        <script src="https://hps.github.io/token/2.1/securesubmit.js" src_type="url" />
+        -->
+        <!-- PROD
+        <script src="https://api2.heartlandportico.com/SecureSubmit.v1/token/2.1/securesubmit.js" src_type="url" />
+        -->
+        <link src="HPS_Heartland/js/view/payment/securesubmit.js" id="HPS_SECURESUBMIT_IFRAME_CODE"/>
```

Comment:
One script uses a `script` tag while the other uses a `link` tag. Is there a typo with one of these, or are both appropriate?

Reply:
Yes shane, both can be apply to call js file. 
Ref link : http://devdocs.magento.com/guides/v2.0/frontend-dev-guide/layouts/xml-manage.html

![script](https://user-images.githubusercontent.com/10886482/37750934-e202876c-2d65-11e8-881f-0e92a260398f.png)

Reply:
Yes shane, both can be apply to call js file. 
Ref link : http://devdocs.magento.com/guides/v2.0/frontend-dev-guide/layouts/xml-manage.html

![script](https://user-images.githubusercontent.com/10886482/37750934-e202876c-2d65-11e8-881f-0e92a260398f.png)

Reply:
Awesome! Thanks for the confirmation!